### DISCUSSION TITLE
2022-07: 重新掌握时间感

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#336] [Announcements]
# 2022-07: 重新掌握时间感

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-07/

**Comment by lilydjwg** [comment]
我的电子钟提升了我对 NTP 的重要性的认知 😂

**Comment by MarvelousBlack** [comment]
直接寫個10分鐘彈窗休息一下，那鍾都可以省掉了呢（

> 自从有了这个钟，我再也没有错过过饭点和下班的时间(哦豁)。

那把所有有時間可以看的地方都去掉，那不就瘋狂加班（老闆狂喜

**Comment by zjuyk** [comment]
规律作息，养成生物钟才是王道

**Comment by hualet** [comment]
上下课这个叫法挺不错 :D

**Comment by yaoyuanyy** [comment]
电子钟贴个图片吧，我同步买个

**Reply by Xuanwo** [reply]
得力计时器提醒学生做题考研定时器厨房多功能时间闹钟秒表倒两用

https://detail.tmall.com/item.htm?id=625595734011
